apps/passport-server/test/zupass.spec.ts

```diff
@@ -126,16 +204,62 @@ describe("zupass functionality", function () {
         throw new Error("couldn't find a resident to test with");
       }
 
-      residentUser = await testLoginZupass(application, resident.email, {
+      const result = await testLoginZupass(application, resident.email, {
         force: false,
         expectAlreadyRegistered: false,
         expectDoesntHaveTicket: false,
         expectEmailInvalid: false
       });
+      residentUser = result?.user;
+      residentIdentity = result?.identity;
+
       expect(emailAPI.send).to.have.been.called.exactly(1);
     }
   );
 
+  step(
+    "after logging in, user should be able to be issued some PCDs from the server",
+    async function () {
```

Comment:
With this test and some of the others in this file, we can get rid of the `zuzalu-ticket-feed.spec.ts` file, since it duplicates what's happening here.

Reply:
I'll be sure to reconcile redundant tests this week!